OpenRA.Mods.Common/Irc.cs

```diff
@@ -0,0 +1,21 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Threading;
+using Meebey.SmartIrc4net;
+
+namespace OpenRA.Mods.Common
+{
+	public static class Irc
+	{
+		public static IrcClient Client;
```

Comment:
Made it `volatile`. I only need `JoinedChannels` once at first to initialize the user list when returning to the widget. In all other cases I use the channel provided by the event args.